lib/LocalizationHelper.js

```diff
@@ -30,68 +30,68 @@
 
 'use strict';
 
-module.exports = LocalizationHelper;
+class LocalizationHelper {
 
-/**
- * Creates new instance of localization helper.
- * @param {LocalizationProvider} $localizationProvider Localization provider.
- * @constructor
- */
-function LocalizationHelper($localizationProvider) {
-	this._localizationProvider = $localizationProvider;
-}
+	/**
+	 * Creates new instance of localization helper.
+	 * @param {ServiceLocator} locator Locator to resolve dependencies.
+	 * @constructor
+	 */
+	constructor(locator) {
+		this._localizationProvider = locator.resolve('localizationProvider');
+	}
 
-/**
- * Current localization provider.
- * @type {LocalizationProvider}
- * @private
- */
-LocalizationHelper.prototype._localizationProvider = null;
+	/**
+	 * Gets handlebars helper for localization.
+	 * @returns {Function} Handlebars helper function.
+	 */
+	getHandlebarsHelper() {
 
-/**
- * Gets handlebars helper for localization.
- * @returns {Function} Handlebars helper function.
- */
-LocalizationHelper.prototype.getHandlebarsHelper = function () {
-	return function () {
-		var key = null,
-			locale = null,
-			count = null,
-			options = arguments[arguments.length - 1];
+		/* eslint prefer-rest-params:0 */
+		/* eslint no-invalid-this:0 */
+		// TODO rewrite this when Spread operator and Rest parameters will be supported
+		return function() {
```

Comment:
We can't use arrow function because we use arguments and in arrow function is empty